src/backend/optimizer/plan/setrefs.c

```diff
@@ -1142,6 +1142,7 @@ set_plan_refs(PlannerInfo *root, Plan *plan, int rtoffset)
 		case T_Agg:
 			{
 				Agg		   *agg = (Agg *) plan;
+				int			context = 0;
```

Comment:
context use heap var is better.

Reply:
Yes, use a int value as context is a bad choice. I am considering change it.